src/parser/view_transform.c

```diff
@@ -8856,21 +8856,14 @@ mq_class_lambda (PARSER_CONTEXT * parser, PT_NODE * statement, PT_NODE * class_,
 	  spec->info.spec.path_entities = NULL;
 	  if (newspec)
 	    {
-	      if (newspec->info.spec.derived_table_type == PT_DERIVED_JSON_TABLE)
+	      if ((newspec->info.spec.derived_table_type == PT_DERIVED_JSON_TABLE)
+		  || (newspec->info.spec.derived_table_type == PT_DERIVED_DBLINK_TABLE))
```

Comment:
is  this routine necessary? Does removing it cause any problems?

Reply:
I'm not even sure if that code must be there.
However, the code was written by imitating the JSON_TABLE processing routine.
We need to look a little more closely at what cases that code is handled in.

Reply:
As a result of testing, it seems that there is no need to separate and process PT_DERIVED_DBLINK_TABLE.
I'll  fix the code.